### ISSUE TITLE
multiple instances of component not firing click-outside

### ISSUE THREAD

**Issue by ultish** [zeppelin/ember-click-outside#115]
Hi,

I have multiple instances of a component that use the click-outside modifier but when I click outside, only the last instance of that component gets triggered.

eg
```
{{#each this.items as |item|}}
   <ItemDisplay @item={{item}} />
{{/each}}

```
And within ItemDisplay's template:
```
<div {{on-click-outside (fn this.clearSelections)}}>
   ...
</div>
```

Now what I see when I click outside of these components is only 1 trigger of the clearSelections action and it's always on the last instance of <ItemDisplay> that's getting the event.

Have I wired this incorrectly or is there an option I'm missing?

**Comment by zeppelin** [comment]
@ultish No, this should be working OK. I'll definetely investigate this weekend!

Not quite related: which version are you using? :)

**Comment by ultish** [comment]
@zeppelin I'm running:
"ember-cli": "~3.17.0"
"ember-click-outside": "^2.0.0-beta.0"

Cheers :)

**Comment by zeppelin** [comment]
@ultish Confirmed, this is definitely a bug. Will fix as soon as possible.

**Comment by zeppelin** [comment]
@ultish Fixed & released as `v2.0.0-beta.1`, thanks for reporting!

Turns out this was a 2.x beta issue after all :)

**Comment by ultish** [comment]
@zeppelin awesome 👍 Thanks for the quick turnaround :)
